aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -51,7 +50,7 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
     protected static final String STACK_NAME = "rds";
     protected static final int MAX_LENGTH_GROUP_NAME = 255;
     // 5 min for waiting propagation according to https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_ModifyDBClusterParameterGroup.html
-    protected static final int NO_CALLBACK_DELAY = 0;
+    protected static final int STABILIZATION_DELAY_SECONDS = 5 * 60;
```

Comment:
The fact that we have to explain what time unit is being used in the var name makes me think we are using a wrong container (`int`). This is an internal property and it won't be serialized/deserialized. Let's use a more semantically-matching class `Duration` here. It could look like:
```java
protected static final Duration STABILIZATION_DELAY = Duration.ofMinutes(5);
```